webpack.config.js

```diff
@@ -89,7 +97,10 @@ module.exports = {
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
 
     new webpack.DefinePlugin({
-        BACKEND_URL: JSON.stringify(process.env.BACKEND_URL || 'http://backend.url.undefined')
+        BACKEND_URL: JSON.stringify(process.env.BACKEND_URL || 'http://backend.url.undefined'),
+        RAVEN_URL: JSON.stringify(process.env.RAVEN_URL || false),
```

Comment:
This moves RAVEN_URL config into .env instead of config.js
I'll need to reconfigure that in ansible